### DISCUSSION TITLE
recipes/breads/ciabatta/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1020] [Comments]
# recipes/breads/ciabatta/

A collection of my recipes.

https://nicholaswilde.io/recipes/breads/ciabatta/

**Comment by nicholaswilde** [comment]
- https://www.youtube.com/watch?v=bRGHpCNl72o
- https://www.youtube.com/watch?v=xEWGKFe21t4
- https://www.kingarthurbaking.com/recipes/rustic-italian-ciabatta-recipe
